UI/src/view/point_asset/pointAssetForm.js

```diff
@@ -274,19 +290,103 @@ root.PointAssetForm = function() {
+        '    </div>';
+  };
+
+  this.singleChoiceHandler = function (property) {
+    var propertyValue = (property.values.length === 0) ? '' : _.head(property.values).propertyValue;
+    var propertyValues = _.head(_.map(_.filter(me.enumeratedPropertyValues, { 'publicId': property.publicId}), function(val) {return val.values; }));
+    var propertyDefaultValue = _.indexOf(_.map(propertyValues, function (prop) {return _.some(prop, function(propValue) {return propValue == propertyValue;});}), true);
+    var selectableValues = _.map(propertyValues, function (label) {
+      return $('<option>',
+          { selected: propertyValue == label.propertyValue,
+            value: parseFloat(label.propertyValue),
+            text: label.propertyDisplayValue}
+      )[0].outerHTML; }).join('');
+    return '' +
+        '    <div class="form-group editable form-point-asset">' +
+        '      <label class="control-label">' + property.localizedName + '</label>' +
+        '      <p class="form-control-static">' + (propertyValues[propertyDefaultValue].propertyDisplayValue || '-') + '</p>' +
+        '      <select class="form-control" data-grouped-id="'+ property.groupedId +'" style="display:none" id=' + property.publicId + (property.groupedId ? ('-' + property.groupedId) : '') +'>' +
+        selectableValues +
+        '      </select>' +
+        '    </div>';
+  };
+
+  this.readOnlyHandler = function (property) {
+    var propertyValue = (property.values.length === 0) ? '' : property.values[0].propertyValue;
+    var displayValue = (property.localizedName) ? property.localizedName : (property.values.length === 0) ? '' : property.values[0].propertyDisplayValue;
+    return '' +
+        '    <div class="form-group editable form-point-asset">' +
+        '        <label class="control-label">' + displayValue + '</label>' +
+        '        <p class="form-control-static">' + propertyValue + '</p>' +
+        '    </div>';
+  };
+
+  this.dateHandler = function(property) {
+    var propertyValue = '';
+
+    if ( !_.isEmpty(property.values) && !_.isEmpty(_.head(property.values).propertyDisplayValue) )
+      propertyValue = _.head(property.values).propertyDisplayValue;
+
+    return '' +
+        '<div class="form-group editable form-point-asset">' +
+        '     <label class="control-label">' + property.localizedName + '</label>' +
+        '     <p class="form-control-static">' + (propertyValue || '–') + '</p>' +
+        '     <input type="text" data-grouped-id="'+ property.groupedId +'" class="form-control" id="' + property.publicId + (property.groupedId ? ('-' + property.groupedId) : '') + '" value="' + propertyValue + '">' +
+        '</div>';
+  };
+
+  this.checkboxHandler = function(property) {
+    var checked = _.head(property.values).propertyValue == "0" ? '' : 'checked';
```

Comment:
Apply same Idea

```js 
 var checked = _.isEmpty( _.head(property.values).propertyValue ) ? '' : 'checked';
```

Reply:
This suggestion is not correct, having **propertyValue == "0"** is different than having **_.isEmpty(propertyValue)**. Zero is an acceptable value